docs/filter-functions.md

```diff
@@ -32,32 +32,33 @@ The following functions are supported in addition to the built-in functions prov
+| [`extractSonarFindings`](#extractsonarfindings) :fontawesome-brands-github: <br />Get an object with a summary of the findings found by the SonarCloud scan | [`pr`](./context-variables.md#pr)                          | -                                                  | Object                  |
+| [`explainRankByGitBlame`](#explainrankbygitblame)<br />Short markdown text explaining rankByGitBlame results                                                | [`repo`](./context-variables.md#repo)                      | `gt`, `lt`                                         | [String]                |
+| [`isFirstCommit`](#isfirstcommit)<br />Checks if its the author first commit in the repo                                                                    | [`repo.contributors`](./context-variables.md#repo)         | String                                             | Bool                    |
+| [`isFormattingChange`](#isformattingchange)<br />Checks that only formatting changed                                                                        | [[`FileDiff` ](./context-variables.md#filediff-structure)] | -                                                  | Bool                    |
+| [`mapToEnum`](#maptoenum)<br />return the enum value matches to the input key                                                                               | String                                                     | Enum object                                        | Object                  |
+| [`matchDiffLines`](#matchdifflines)<br />Match every line in diff                                                                                           | [[`FileDiff` ](./context-variables.md#filediff-structure)] | `regex`, `ignoreWhiteSpaces`                       | [Bool]                  |
+| [`rankByGitActivity`](#rankbygitactivity)<br />Get list of contributors based on `git-commit` activity                                                      | [`repo`](./context-variables.md#repo)                      | `gt`, `lt`                                         | [String]                |
+| [`rankByGitBlame`](#rankbygitblame)<br />Get list of contributors based on `git-blame` results                                                              | [`repo`](./context-variables.md#repo)                      | `gt`, `lt`                                         | [String]                |
+| [`readFile`](#readfile)<br />Reads the contents of a file from the current branch or "cm" folder                                                            | String - The file path                                     | String (optional) - the file type. `txt` or `json` | String                  |
```

Comment:
It's not clear "from the current branch or cm folder", is it the current branch? what path? is it relative path? to what?